base/essentials.jl

```diff
@@ -158,6 +158,19 @@ macro boundscheck(blk)
       $(Expr(:boundscheck,:pop)))
 end
 
+"""
+    @inbounds(blk)
+
+In the example below the bound check of array A is skipped to improve performance.
```

Comment:
Should probably add a warning to this docstring that using `@inbounds` may return incorrect results/clashes/corruption for out-of-bounds indices and that the user is responsible for checking manually.